src/TheClassLessTheme/Views/Layout.liquid

```diff
@@ -1,51 +1,46 @@
-<!doctype html>
-<html lang="{{ Culture.Name }}">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{ Site.SiteName }}</title>
-  <!-- 
-<link rel="stylesheet" href="https://unpkg.com/awsm.css/dist/awsm.min.css">
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/holiday.css"/>
-<link rel="stylesheet" href="https://unpkg.com/marx-css/css/marx.min.css">
-<link rel="stylesheet" href="https://unpkg.com/mvp.css">
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/kognise/water.css@latest/dist/dark.min.css">
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/kognise/water.css@latest/dist/light.min.css">
-	<link rel="stylesheet" href="{{ '~/TheClassLessTheme/css/marx.min.css' | href }}">
-	<link rel="stylesheet" href="https://mblode.github.io/marx/stylesheets/all-3e842764.css">-->
-		<link rel="stylesheet" href="https://unpkg.com/sakura.css/css/sakura.css" type="text/css">
+<!DOCTYPE html>
+<html lang="{{ Culture.Name }}" data-theme="{{ Site.Properties.ThemeSettings.ThemeSettings.ThemeVariation.Text }}">
+    <head>
+        <meta charset="utf-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>{{ Site.SiteName }}</title>
+        {% link type:"image/x-icon", rel:"shortcut icon", href:"~/TheClassLessTheme/favicon.ico" %}
+        <!-- 
+        <link rel="stylesheet" href="https://unpkg.com/awsm.css/dist/awsm.min.css">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/holiday.css"/>
+        <link rel="stylesheet" href="https://unpkg.com/marx-css/css/marx.min.css">
+        <link rel="stylesheet" href="https://unpkg.com/mvp.css">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/kognise/water.css@latest/dist/dark.min.css">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/kognise/water.css@latest/dist/light.min.css">
+	    <link rel="stylesheet" href="{{ '~/TheClassLessTheme/css/marx.min.css' | href }}">
+	    <link rel="stylesheet" href="https://mblode.github.io/marx/stylesheets/all-3e842764.css">-->
+      
+        {% if Site.Properties.ThemeSettings.ThemeSettings.UseTinyVersion.Value %}
+            <link rel="stylesheet" href="https://classless.de/classless-tiny.css">
+        {% else %}
+            <link rel="stylesheet" href="https://classless.de/classless.css"/>
+        {% endif %}
+        <link rel="stylesheet" href="https://classless.de/addons/themes.css"/>
```

Comment:
On https://classless.de, under "Getting Started" it reads (emphasis mine):

> For testing, you can simply insert the following line into your HTML file. **For production, please host the classless.css file yourself.**

Thus, this theme should contain the files by itself, not hotlink them.